File: app/src/main/java/com/giphy/browser/model/Pagination.java
package com.giphy.browser.model;

import com.squareup.moshi.Json;

public class Pagination {
    @Json(name = "offset")
    public int offset = 0;

    @Json(name = "total_count")
    public int totalCount = 0;

    @Json(name = "count")
    public int count = 0;
}

File: app/src/main/java/com/giphy/browser/common/model/Pagination.kt
package com.giphy.browser.common.model

import com.squareup.moshi.Json

data class Pagination(
    @field:Json(name = "offset") var offset: Int = 0,
    @field:Json(name = "total_count") var totalCount: Int = 0,
    @field:Json(name = "count") var count: Int = 0
) {
    fun nextOffset(): Int = offset + count
}
